tpm2/tpm2.go

```diff
@@ -418,6 +418,57 @@ func CreatePrimary(rw io.ReadWriter, owner tpmutil.Handle, sel PCRSelection, par
 	return decodeCreatePrimary(resp)
 }
 
+// CreatePrimaryEx initializes the primary key in a given hierarchy.
+// This function differs from CreatePrimary in that all response elements
+// are returned, and they are returned in relatively raw form.
+//
+// The first return value is a transient handle to the loaded key.
```

Comment:
Use named return values and refer to them by the name instead of position.
Values with obvious names can skip a doc sentence.